scripts/create-release-branch.js

```diff
@@ -92,12 +89,8 @@ shell.exec('git add lerna.json');
 // Git commit
 shell.exec(`git commit -m "chore: update to version ${nextVersion}"`);
 
-// Merge release branch to develop as soon as it is cut.
-// In this way, we can resolve conflicts between main branch and develop branch when merge main back to develop after the release.
-shell.exec(`git checkout develop`)
-shell.exec(`git merge ${releaseBranchName}`)
-shell.exec(`git push -u origin develop`)
-shell.exec(`git checkout ${releaseBranchName}`)
+// Create the new release branch and switch to it
+shell.exec(`git checkout -b ${releaseBranchName}`);
 
 // Generate changelog
 const previousBranchName = changeLogGeneratorUtils.getPreviousReleaseBranch(releaseBranchName);
```

Comment:
I'm still a bit confused about the flow here.  I made this drawing to show the end result.

Reply:
![releaseGitBranches](https://user-images.githubusercontent.com/76090802/166569272-dc91e9a6-682e-4758-8458-e4da4fae3ed6.png)

Reply:
Do we need a push back to the remote develop after the version commit in order to avoid the merge conflicts?  Also I think we might have an issue committing directly to develop.  It's a protected branch that requires PRs for merge (but there may be exceptions for the release user I'm not sure)